src/components/appSwitch/AppSwitch.js

```diff
@@ -261,7 +254,7 @@ const AppSwitch = (props: LoadingProps) => {
     ) {
       return
     }
-    userStorage.userProperties.set('lastBonusCheckDate', new Date().toISOString())
+    await userStorage.userProperties.set('lastBonusCheckDate', new Date().toISOString())
     await checkBonusesToRedeem()
```

Comment:
do we need to await checkBonusesToRedeem ?

probably we could do like that ?
```suggestion
    const { userProperties } = userStorage
    const now = new Date().toISOString() // those both definitions could be moved to the top of the init function
        
    userProperties.set('lastBonusCheckDate', now).then(checkBonusesToRedeem)
```

Reply:
nothing is awaiting on the parent method so there's no need